src/test/java/com/commercetools/sync/sdk2/types/utils/FieldDefinitionUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,500 @@
+package com.commercetools.sync.sdk2.types.utils;
+
+import static com.commercetools.api.models.common.LocalizedString.ofEnglish;
+import static com.commercetools.sync.sdk2.types.utils.FieldDefinitionFixtures.stringFieldDefinition;
+import static com.commercetools.sync.sdk2.types.utils.FieldDefinitionUpdateActionUtils.buildActions;
+import static com.commercetools.sync.sdk2.types.utils.FieldDefinitionUpdateActionUtils.buildChangeLabelUpdateAction;
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.commercetools.api.models.type.CustomFieldEnumTypeBuilder;
+import com.commercetools.api.models.type.CustomFieldEnumValue;
+import com.commercetools.api.models.type.CustomFieldEnumValueBuilder;
+import com.commercetools.api.models.type.CustomFieldLocalizedEnumTypeBuilder;
+import com.commercetools.api.models.type.CustomFieldLocalizedEnumValue;
+import com.commercetools.api.models.type.CustomFieldLocalizedEnumValueBuilder;
+import com.commercetools.api.models.type.FieldDefinition;
+import com.commercetools.api.models.type.FieldDefinitionBuilder;
+import com.commercetools.api.models.type.TypeAddEnumValueActionBuilder;
+import com.commercetools.api.models.type.TypeAddLocalizedEnumValueActionBuilder;
+import com.commercetools.api.models.type.TypeChangeEnumValueOrderActionBuilder;
+import com.commercetools.api.models.type.TypeChangeLabelActionBuilder;
+import com.commercetools.api.models.type.TypeChangeLocalizedEnumValueOrderActionBuilder;
+import com.commercetools.api.models.type.TypeTextInputHint;
+import com.commercetools.api.models.type.TypeUpdateAction;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FieldDefinitionUpdateActionUtilsTest {
+  private static final String FIELD_NAME_1 = "fieldName1";
+  private static final String LABEL_1 = "label1";
+  private static final String LABEL_2 = "label2";
+
+  private static FieldDefinition old;
+  private static FieldDefinition newSame;
+  private static FieldDefinition newDifferent;
+
+  private static final CustomFieldEnumValue ENUM_VALUE_A =
+      CustomFieldEnumValueBuilder.of().key("a").label("label_a").build();
+  private static final CustomFieldEnumValue ENUM_VALUE_B =
+      CustomFieldEnumValueBuilder.of().key("b").label("label_b").build();
+
+  private static final CustomFieldLocalizedEnumValue LOCALIZED_ENUM_VALUE_A =
+      CustomFieldLocalizedEnumValueBuilder.of().key("a").label(ofEnglish("label_a")).build();
+  private static final CustomFieldLocalizedEnumValue LOCALIZED_ENUM_VALUE_B =
+      CustomFieldLocalizedEnumValueBuilder.of().key("b").label(ofEnglish("label_b")).build();
```

Comment:
Use PlainEnumValueFixtures.CustomFieldEnumValueFixtures and LocalizedEnumValueFixures.CustomFieldLocalizedEnumValueFixtures from the common/utils package

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/1007/commits/a3c40cea6304f8f6ad06a9f11ec472dbd76490f5